compiler/noirc_frontend/src/hir/def_collector/errors.rs

```diff
@@ -77,9 +77,14 @@ pub struct MacroError {
 }
 
 impl DefCollectorErrorKind {
-    pub fn into_file_diagnostic(self, file: fm::FileId) -> FileDiagnostic {
+    pub fn into_file_diagnostic(&self, file: fm::FileId) -> FileDiagnostic {
         Diagnostic::from(self).in_file(file)
     }
+
+    pub fn is_error(&self) -> bool {
+        // All errors are warnings currently
```

Comment:
All warnings are errors?

Reply:
Right, whoops

Reply:
I just removed it since the function is unused